x-pack/legacy/plugins/apm/public/components/shared/TransactionBreakdown/TransactionBreakdownGraph/index.tsx

```diff
@@ -4,10 +4,62 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
+import React, { useMemo, useCallback } from 'react';
+import numeral from '@elastic/numeral';
+import { NOT_AVAILABLE_LABEL } from '../../../../../common/i18n';
+import { Coordinate } from '../../../../../typings/timeseries';
+import { TransactionLineChart } from '../../charts/TransactionCharts/TransactionLineChart';
+import { asPercent } from '../../../../utils/formatters';
+import { unit } from '../../../../style/variables';
 
-const TransactionBreakdownGraph: React.FC<{}> = () => {
-  return <div />;
+interface Props {
+  timeseries: Array<{
+    name: string;
+    color: string;
+    values: Array<{ x: number; y: number | null }>;
+  }>;
+}
+
+const TransactionBreakdownGraph: React.FC<Props> = props => {
+  const { timeseries } = props;
+
+  const series: React.ComponentProps<
+    typeof TransactionLineChart
+  >['series'] = useMemo(
+    () => {
+      return timeseries.map(timeseriesConfig => {
+        return {
+          title: timeseriesConfig.name,
```

Comment:
Why is this rename necessary? Shouldn't `timeseriesConfig.name` be `timeseriesConfig.title` in the first place? Same with `timeseriesConfig.values` - shouldn't that be `timeseriesConfig.data` in the first place?

I actually prefer `values` over `data` but then we should change that everywhere.